tests/unit/conftest.py

```diff
@@ -36,3 +36,62 @@ def mock_airflow_connection(mocker, dr_conn_details):
         ),
     )
     mocker.patch.dict("os.environ", AIRFLOW_CONN_DATAROBOT_DEFAULT=conn.get_uri())
+
+
+# For JDBC Connection
+@pytest.fixture
+def dr_jdbc_conn_details():
+    return {
+        "login": "test_login",
+        "password": "test_password",
+        "datarobot_connection": "datarobot_default",
+        "jdbc_driver": "test JDBC Driver",
+        "jdbc_url": "jdbc:test_jdbc_connection_string",
+    }
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
nit: `scope="function"` is the default, so no need to mention it explicitely

Reply:
```suggestion
@pytest.fixture(autouse=True)
```